src/test/kotlin/io/github/mojira/arisa/modules/ConfirmParentModuleTest.kt

```diff
@@ -0,0 +1,90 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.right
+import io.kotest.assertions.arrow.either.shouldBeLeft
+import io.kotest.assertions.arrow.either.shouldBeRight
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class ConfirmParentModuleTest : StringSpec({
+    "should return OperationNotNeededModuleResponse when Linked is null" {
+        val module = ConfirmParentModule(listOf("Unconfirmed", "Plausible"), "Community Consensus")
+        val request = ConfirmParentModule.Request("Unconfirmed", null) { Unit.right() }
+
+        val result = module(request)
+
+        result.shouldBeLeft(OperationNotNeededModuleResponse)
+    }
+
+    "should return OperationNotNeededModuleResponse when Linked is 0" {
+        val module = ConfirmParentModule(listOf("Unconfirmed", "Plausible"), "Community Consensus")
+        val request = ConfirmParentModule.Request("Unconfirmed", 0.0) { Unit.right() }
+
+        val result = module(request)
+
+        result.shouldBeLeft(OperationNotNeededModuleResponse)
+    }
+
+    "should set to Community Consensus when Confirmation Status is unset and Linked is 1" {
+        var changedConfirmation = ""
+
+        val module = ConfirmParentModule(listOf("Unconfirmed", "Plausible"), "Community Consensus")
+        val request = ConfirmParentModule.Request(null, 1.0) {
```

Comment:
It's also sometimes the empty string, not sure about the exact conditions, but empty string should also be counted as "Unconfirmed"